tomviz/python/tomviz/threaded_dataset.py

```diff
@@ -0,0 +1,54 @@
+from tomviz import utils
+import vtk.util.numpy_support as np_s
+
+
+class Dataset:
+    def __init__(self, data_object, data_source):
+        self._data_object = data_object
+        self._data_source = data_source
+
+        self._tilt_axis = 2
+
+    @property
+    def active_scalars(self):
+        return utils.get_array(self._data_object)
+
+    @active_scalars.setter
+    def active_scalars(self, v):
+        utils.set_array(self._data_object, v)
+
+    @property
+    def scalars(self):
+        return { key: value for key, value in utils.arrays(self._data_object) }
```

Comment:
This is the only part of the API that I am not sure about. We have always used scalars to mean a single component.  `scalars` here is referring to multiple components were a `active_scalars` is referring to a single component.

Reply:
@cryos Do you have any thoughts?

Reply:
We could potentially rename `scalars` to be `all_scalars`, and then rename `active_scalars` to just `scalars`, which is similar to VTK (`GetScalars` in VTK returns the active scalars if no name is provided).

There isn't really an equivalent in VTK for `all_scalars`, though.

Reply:
We resolved this through conversation.